ext/symengine/symengine.c

```diff
@@ -46,4 +47,10 @@ void Init_symengine() {
     c_rational = rb_define_class_under(m_symengine, "Rational", c_basic);
     rb_define_alloc_func(c_rational, cbasic_alloc);
     rb_define_method(c_rational, "initialize", crational_init, 1);
+
+    //Constants
+    c_constants = rb_define_class_under(m_symengine, "Constants", c_basic);
```

Comment:
This should be singular as in `Constant`. While comparing this sounds intuitive
```ruby
SymEngine::E.instance_of? SymEngine::Constant
```
not `SymEngine::Constants`